docs/included-feature-flags.md

```diff
@@ -20,4 +20,8 @@ message will be displayed when the admin level is changed.
 
 ## AllowChatCommandExecuting (default: `true`)
 If true, commands can be executed from the chat with
-prefixes used. If false, chat executing will be ignored.
\ No newline at end of file
+prefixes used. If false, chat executing will be ignored.
+
+## AllowDroppingSwords (default: `false`)
```

Comment:
The wrong default value was noted. It actually defaults to `true` to be backward compatible.
https://github.com/TheNexusAvenger/Nexus-Admin/blob/9ab894f712c18df9eab0a421027dd0eda5b63ca5/src/IncludedCommands/BasicCommands/sword.lua#L26

Reply:
Done.